recipes/xkbcommon/all/conanfile.py

```diff
@@ -63,20 +63,31 @@ def requirements(self):
         if self.options.get_safe("xkbregistry"):
             self.requires("libxml2/2.9.12")
         if self.options.with_wayland:
-            # FIXME: In cross-compiling scenarios this library requires 'wayland' in _host_ to use and link some
-            # FIXME:    sources, but it requires tools like 'wayland-scanner' in the _build_ context. Probably
-            # FIXME:    'wayland-protocols' should also be a build-requires as it is providing just some files
-            # FIXME:    used during the build process.
             self.requires("wayland/1.19.0")
-            self.requires("wayland-protocols/1.21")
+            self.requires("wayland-protocols/1.21")  # FIXME: This should be a build-requires
```

Comment:
@jgsogo do you remember why you put this as a fixme, rather than directly adding it to build _requirements() ?

Reply:
Not really 😞 . 

Probably because of how the information is propagated in Conan. This `-protocols` should contain something only used during the build (some files), and probably if declared as a build-requires it is harder to consume them.

In any case, it would require a new PR and some investigation